{
    "MITRE": {
        "CVE_data_meta": {
            "ASSIGNER": "psirt@us.ibm.com",
            "DATE_PUBLIC": "2021-10-14T00:00:00",
            "ID": "CVE-2021-29679",
            "STATE": "PUBLIC"
        },
        "affects": {
            "vendor": {
                "vendor_data": [
                    {
                        "product": {
                            "product_data": [
                                {
                                    "product_name": "Cognos Analytics",
                                    "version": {
                                        "version_data": [
                                            {
                                                "version_value": "11.2.0"
                                            },
                                            {
                                                "version_value": "11.1.7"
                                            }
                                        ]
                                    }
                                }
                            ]
                        },
                        "vendor_name": "IBM"
                    }
                ]
            }
        },
        "data_format": "MITRE",
        "data_type": "CVE",
        "data_version": "4.0",
        "description": {
            "description_data": [
                {
                    "lang": "eng",
                    "value": "IBM Cognos Analytics 11.1.7 and 11.2.0 could allow an authenticated user to execute code remotely due to incorrectly neutralizaing user-contrlled input that could be interpreted a a server-side include (SSI) directive. IBM X-Force ID: 199915."
                }
            ]
        },
        "impact": {
            "cvssv3": {
                "BM": {
                    "A": "H",
                    "AC": "L",
                    "AV": "N",
                    "C": "H",
                    "I": "H",
                    "PR": "L",
                    "S": "U",
                    "SCORE": "8.800",
                    "UI": "N"
                },
                "TM": {
                    "E": "U",
                    "RC": "C",
                    "RL": "O"
                }
            }
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": [
                        {
                            "lang": "eng",
                            "value": "Gain Privileges"
                        }
                    ]
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "name": "https://www.ibm.com/support/pages/node/6491661",
                    "refsource": "CONFIRM",
                    "title": "IBM Security Bulletin 6491661 (Cognos Analytics)",
                    "url": "https://www.ibm.com/support/pages/node/6491661"
                },
                {
                    "name": "ibm-cognos-cve202129679-code-exec (199915)",
                    "refsource": "XF",
                    "title": "X-Force Vulnerability Report",
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/199915"
                }
            ]
        }
    },
    "NVD": {
        "configurations": {
            "CVE_data_version": "4.0",
            "nodes": []
        },
        "cve": {
            "CVE_data_meta": {
                "ASSIGNER": "psirt@us.ibm.com",
                "ID": "CVE-2021-29679"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "en",
                        "value": "IBM Cognos Analytics 11.1.7 and 11.2.0 could allow an authenticated user to execute code remotely due to incorrectly neutralizaing user-contrlled input that could be interpreted a a server-side include (SSI) directive. IBM X-Force ID: 199915."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": []
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://www.ibm.com/support/pages/node/6491661",
                        "refsource": "CONFIRM",
                        "tags": [],
                        "url": "https://www.ibm.com/support/pages/node/6491661"
                    },
                    {
                        "name": "ibm-cognos-cve202129679-code-exec (199915)",
                        "refsource": "XF",
                        "tags": [],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/199915"
                    }
                ]
            }
        },
        "impact": {},
        "lastModifiedDate": "2021-10-15T16:27Z",
        "publishedDate": "2021-10-15T16:15Z"
    }
}